ting_openuserstatus.module

```diff
@@ -190,4 +190,4 @@ function ting_openuserstatus_do_request($params) {
     }
   }
   return $params;
-}
\ No newline at end of file
+}
```

Comment:
måske du også skulle omdøbe filen: ting_openuserstatus.request.inc -> OpenUserstatusRequest.php 
nu når du er igang?

Reply:
Det var ikke nogen dum ide.

Reply:
godkendt - thumbs up